File: CS/DocumentMerger/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DocumentMerger.Helpers;
using DevExpress.XtraRichEdit.API.Native;

namespace DocumentMerger {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            List<string> filenames = new List<string>() {  
                 @"Documents\FloatingObjects.rtf", 
                 @"Documents\CharacterFormatting.rtf",
                 @"Documents\HeadersFooters.rtf"
            };

            Document mergedDoc = DocumentsMerger.MergeDouments(filenames);
            richEditControl1.RtfText = mergedDoc.RtfText;
        }
    }
}

File: VB/DocumentMerger/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DocumentMerger.Helpers
Imports DevExpress.XtraRichEdit.API.Native

Namespace DocumentMerger

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim filenames As List(Of String) = New List(Of String)() From {"Documents\FloatingObjects.rtf", "Documents\CharacterFormatting.rtf", "Documents\HeadersFooters.rtf"}
            Dim mergedDoc As Document = DocumentsMerger.MergeDouments(filenames)
            richEditControl1.RtfText = mergedDoc.RtfText
        End Sub
    End Class
End Namespace
